src/Http/Role/AddLabelToRoleRequestHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\ReadRepositoryInterface;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Role\Commands\AddLabel;
+use Fig\Http\Message\StatusCodeInterface;
+use InvalidArgumentException;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Slim\Psr7\Response;
+
+class AddLabelToRoleRequestHandler implements RequestHandlerInterface
```

Comment:
Nit: We try to make every class either `final` or `abstract`. We don't have a rule for that in the coding standards yet because we still have a lot of classes that are extended but cannot be made abstract without a refactor.

You can add it to your PHP templates in PHPStorm so it gets added automatically if you want.

Reply:
Done.